docs/3.0-migration-guide.d/modify_SHA384_option_behaviour.md

```diff
@@ -0,0 +1,9 @@
+Replaced MBEDTLS_SHA512_NO_SHA384 with MBEDTLS_SHA384_C
+------------------------------------------------------
+
+This does not affect users who use the default `config.h`.
+MBEDTLS_SHA512_NO_SHA384 was disabled by default, now MBEDTLS_SHA384_C is
+enabled by default.
+
```

Comment:
I just noticed something important is missing: users who had their own configuration file with `MBEDTLS_SHA512_C` enabled and not `MBEDTLS_SHA512_NO_SHA384`, and who need SHA-384, now need to explicitly enable `MBEDTLS_SHA384_C`.